### ISSUE TITLE
stats.req.path may not be the real path requested

### ISSUE THREAD

**Issue by dotchev** [watson/request-stats#4]
main.js
```js
'use strict';

const express = require('express');
const requestStats = require('request-stats');

var app = express();

app.use('/api', (req, res, next) => {
  console.log('req.url:', req.url);
  console.log('req.originalUrl:', req.originalUrl);
  res.end('OK');
});

var server = app.listen(process.env.PORT || 3000, () => {
  console.log('Listening on port %d', server.address().port);
});

requestStats(server, (stats) => {
  console.log('stats.req.path:', stats.req.path);
});
```
```sh
$ node main.js 
Listening on port 3000
```
```sh
$ curl http://localhost:3000/api/user
```
```
req.url: /user
req.originalUrl: /api/user
stats.req.path: /user
```

**Comment by watson** [comment]
Hmm it seems that express is modifying `req.url` property 😞 Not the best move by express IMHO, so I'm not really sure if there's much I can do about it.

I could of course use `req.originalUrl || req.url`, but that seems oddly specific for this use-case. And I hate hacks like that. Preferably I'd like to make this more low-level.

How would you feel about just exposing the original request object, so that you can do `stats.req.raw.originalUrl` your self?

**Comment by dotchev** [comment]
Yes, this would be useful as the raw request may contain additional properties of statistical value.
To be consistent, you can expose also the raw response object.

**Comment by watson** [comment]
@dotchev this have now been released in v2.0.0 on npm
